web-animations-1/Overview.bs

```diff
@@ -2465,123 +2465,203 @@ Animation effects {#animation-effects}
+              and rises linearly,
+              but restarts at zero at the start of every new iteration,
+              and remains at its halfway point from the end time onward.">
     <figcaption>
-      Examples of the effect of the [=start delay=]
-      on the endpoints of the [=active interval=].<br>
-      (a) An animation effect with no delay;
-      the [=start time=] and beginning of the [=active interval=]
-      are coincident.<br>
-      (b) An animation effect with a positive delay;
-      the beginning of the [=active interval=]
-      is deferred by the delay.<br>
-      (c) An animation effect with a negative delay;
-      the beginning of the [=active interval=]
-      is brought forward by the delay.
+      A comparison of [=local time=], [=active time=], and [=iteration time=]
+      for an animation with an [=iteration duration=] of 1s
+      and an [=iteration count=] of 2.5.
+      The [=animation/start time=] defined by the [=animation=]
+      and [=animation effect/end time=] defined by the [=animation effect=]
+      are also annotated.
     </figcaption>
   </figure>
 
-  An [=end delay=] can also be specified
-  but is primarily only of use when sequencing animations.
+  In addition to these time spaces
+  we can also refer to the <dfn noexport>document time space</dfn>,
+  which is time space of the [=time values=]
+  of the [=default document timeline=] of the {{Document}}
+  of the [=current global object=].
 
+  Note: While the time spaces themselves are not bounded,
+  Web Animations defines [=active time=] and the [=iteration progress=]
+  such that they are clamped to a set range as shown in the diagram.
+  For example, whilst a time of -1 second
+  is a valid time in <em>active time space</em>,
+  the procedure for calculating the [=active time=]
+  defined in [[#calculating-the-active-time]]
+  will never return a negative value.
 </div>
 
-  [=Animation effects=] define an <dfn>active interval</dfn>
+### The active interval, duration, and delays ### {#the-active-interval}
+
+  [=Animation effects=] define an <dfn>active interval</dfn>,
   which is the period of time during which the effect
-  is scheduled to produce its effect--
-  with the exception of [=fill modes=],
-  which apply outside the [=active interval=].
+  is scheduled to produce its effect
+  (excepting [=fill modes=],
+  which apply outside the [=active interval=]).
+  Each [=animation effect=] has only one such interval.
 
   The lower bound of the [=active interval=]
-  is defined by the [=start delay=].
-
-  The <dfn>start delay</dfn> of an [=animation effect=]
-  is a signed offset from the [=start time=] of the [=animation=]
-  with which the animation effect is associated.
+  by default corresponds to the [=start time=] of the [=associated animation=],
+  but can be shifted by the <dfn>start delay</dfn>,
+  which is a signed offset from the [=start time=] of the [=animation=].
```

Comment:
The original definition here is not quite right. We really should be defining the active interval in terms of local time (since nested effects have active intervals determined by _their_ local time).

So I perhaps something like: "The lower bound of the active interval is defined by the start delay which is a value in local time space defaulting to zero."

Reply:
What about just:

> The lower bound of the animation interval is determined by its start delay which defaults to zero.

The normative definition of how all that works is the calculation of [active time](https://drafts.csswg.org/web-animations-1/#active-time).

If we really want to make it more descriptive, we could add an (effectively non-normative) explanation saying something like: "When the start delay is zero, the lower bound of the active interval coincides with the start time of the associated animation." or something to that effect. Then we'd simply drop that sentence in level 2.

Reply:
We'd need to introduce a concept of the animation effect start time (distinct from the animation start time) for Level 2. I think we *should* do that, but how about tackling that in a separate PR?

For reference, the text that's in this PR is

> The lower bound of the active interval by default corresponds to the start time of the associated animation, but can be shifted by the start delay, which is a signed offset from the start time of the animation.

And it is consolidated from the existing sentences:

> The lower bound of the active interval is defined by the start delay.
>
> The start delay of an animation effect is a signed offset from the start time of the animation with which the animation effect is associated.
> 
> The lower bound of the active interval typically corresponds to the start time of the animation associated with this animation effect but may be shifted by a start delay on the animation effect.

Reply:
Ok, let's go with what you have and fix it in level 2.